### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -859,25 +859,25 @@
   "mapStyleTooltip": "Select map style",
   "mapZoomInTooltip": "Zoom in",
   "mapZoomOutTooltip": "Zoom out",
-  "mapPointNorthUpTooltip": "Point north up",
+  "mapPointNorthUpTooltip": "North upwards",
   "mapAttributionOsmHot": "Map data © [OpenStreetMap](https://www.openstreetmap.org/copyright) contributors • Tiles by [HOT](https://www.hotosm.org/) • Hosted by [OSM France](https://openstreetmap.fr/)",
-  "mapAttributionStamen": "Map data © [OpenStreetMap](https://www.openstreetmap.org/copyright) contributors • Tiles by [Stamen Design](http://stamen.com), [CC BY 3.0](http://creativecommons.org/licenses/by/3.0)",
-  "openMapPageTooltip": "View on Map page",
+  "mapAttributionStamen": "Map data © [OpenStreetMap](https://www.openstreetmap.org/copyright) contributors • Tiles by [Stamen Design](https://stamen.com), [CC BY 3.0](https://creativecommons.org/licenses/by/3.0)",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:864]: CC-BY-SA-4.0 is the latest one. Couldn't find it on the website.

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -231,29 +231,29 @@
...
     }
   },
-  "storageAccessDialogMessage": "Please select the {directory} of “{volume}” in the next screen to give this app access to it.",
+  "storageAccessDialogMessage": "Please select the {directory} of “{volume}” next to give this app access to it.",
   "@storageAccessDialogMessage": {
     "placeholders": {
       "directory": {
         "type": "String",
-        "description": "the name of a directory, using the output of `rootDirectoryDescription` or `otherDirectoryDescription`"
+        "description": "the full name of a folder, using the output of `rootDirectoryDescription` or `otherDirectoryDescription`"
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:256]: A bit awkward for experienced users, but this allows doing away with "directory" entirely.

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -57,10 +57,10 @@
   "saveTooltip": "Save",
   "pickTooltip": "Pick",
 
-  "doubleBackExitMessage": "Tap “back” again to exit.",
+  "doubleBackExitMessage": "Tap “Back” again to exit.",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:60]: Button, or "←"?

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -116,9 +116,9 @@
   "slideshowActionResume": "Resume",
   "slideshowActionShowInCollection": "Show in Collection",
 
-  "entryInfoActionEditDate": "Edit date & time",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:119]: Wrong use of an ampersand

File: lib/l10n/app_en.arb [hunk 5]
```diff
@@ -174,17 +174,17 @@
   "videoLoopModeAlways": "Always",
 
   "videoControlsPlay": "Play",
-  "videoControlsPlaySeek": "Play & seek backward/forward",
+  "videoControlsPlaySeek": "Play and seek backward/forward",
   "videoControlsPlayOutside": "Open with other player",
   "videoControlsNone": "None",
 
   "mapStyleGoogleNormal": "Google Maps",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:181]:
```suggestion
  "mapStyleGoogleNormal": "Google Maps (enemy map)",
```

File: lib/l10n/app_en.arb [hunk 6]
```diff
@@ -456,9 +456,9 @@
 
   "aboutBugSectionTitle": "Bug Report",
   "aboutBugSaveLogInstruction": "Save app logs to a file",
-  "aboutBugCopyInfoInstruction": "Copy system information",
+  "aboutBugCopyInfoInstruction": "Copy system info",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:459]: Shorter

File: lib/l10n/app_en.arb [hunk 7]
```diff
@@ -800,12 +800,12 @@
   "settingsDisplaySectionTitle": "Display",
   "settingsThemeBrightnessTile": "Theme",
   "settingsThemeBrightnessDialogTitle": "Theme",
-  "settingsThemeColorHighlights": "Color highlights",
+  "settingsThemeColorHighlights": "Colored highlights",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:803]: Yes?

File: lib/l10n/app_en.arb [hunk 8]
```diff
@@ -57,10 +57,10 @@
   "saveTooltip": "Save",
   "pickTooltip": "Pick",
 
-  "doubleBackExitMessage": "Tap “back” again to exit.",
+  "doubleBackExitMessage": "Tap “Back” again to exit.",
   "doNotAskAgain": "Do not ask again",
 
-  "sourceStateLoading": "Loading",
+  "sourceStateLoading": "Loading…",
   "sourceStateCataloguing": "Cataloguing",
```
Comment (comradekingu) [inline lib/l10n/app_en.arb:64]: Looks chiefly British, whereas other strings are "color" (US).

**Comment by deckerst** [comment]
https://github.com/deckerst/aves#code

**Pull request closed without merging**